qiita_db/analysis.py

```diff
@@ -79,6 +79,26 @@ def _status_setter_checks(self, conn_handler):
             raise QiitaDBStatusError("Can't set status away from public!")
 
     @classmethod
+    def get_user_default(cls, user):
```

Comment:
This allows you to return the actual object already instantiated, as opposed to just the ID and instantiating, saving a DB call. Do we care about the extra DB call?